Sources/ManagedAppConfigLib/AppConfig.swift

```diff
@@ -13,7 +13,7 @@ import SwiftUI
 /// declaration.
 ///
 /// When the Managed App Config value changes, SwiftUI updates the parts of any view that depend on those properties.
-@available(macOS 11, iOS 13.0, tvOS 13.0, *)
+@available(macOS 11, iOS 14.0, tvOS 13.0, *)
```

Comment:
Thanks for catching this!  It turns out that `@StateObject` is also available in tvOS 14.0+ so that should be updated as well.

```suggestion
@available(macOS 11, iOS 14.0, tvOS 14.0, *)
```

Reply:
Thanks for the confirmation.
I have also updated the pull request title to match the changes.